specs/src/specs/namespace.md

```diff
@@ -94,7 +94,7 @@ Among the potential consequences is the _Woods Attack_, as elaborated in this fo
 
 ## Implementation
 
-See [go-square/namespace](https://github.com/celestiaorg/go-square/tree/main/namespace).
+See [go-square/namespace](https://github.com/celestiaorg/go-square/blob/main/shares/namespace.go).
```

Comment:
@rootulp Can you please confirm whether this is the correct link?

Reply:
I fixed it slightly differently here: https://github.com/celestiaorg/celestia-app/pull/3635/commits/62c3cd6eb16e02a023da6b688f2d3320925feee3 by using a permalink to go-square/namespace (basically the copy of code before the refactor). What you've done here also works.